### ISSUE TITLE
Flow files generation

### ISSUE THREAD

**Issue by thanos91** [cityflow-project/CityFlow#114]
Hi there,

how you created the flow.json files used in your papers.
Is there a python module that automatically converts SUMO demand files to flow.json files?

**Comment by greeneley** [comment]
Me too. I also have problem like you. Cityflow don't indicate us this problem so we don't know how convert SUMO roadnet files into CityFlow format using tools/Converter/converter.py

**Comment by thanos91** [comment]
Yes, this framework was made by the researcher team to produce a handful of
papers. No consideration was given to help the community use this work to
produce derivative works. No documentation was given about
1) how to convert the sumo route files to cityflow format
2) in which format should be the json files for their visualiser and so on
